internal/rbd/nodeserver.go

```diff
@@ -824,7 +828,8 @@ func (ns *NodeServer) NodeUnstageVolume(
 		imgInfo.NbdAccess,
 		imgInfo.Encrypted,
 		req.GetVolumeId(),
-		imgInfo.UnmapOptions); err != nil {
+		imgInfo.UnmapOptions,
```

Comment:
Maybe as a cleanup:
Looks like the number of arguments of this function is growing. can we add detachRBDImageOrDeviceSpec as method to `imgInfo`? or make `imgInfo` as an argument instead of passing individual parameters?

Reply:
There are also other callers of detachRBDImageOrDeviceSpec() which doesn't use imgInfo. Hence it wouldn't be ideal.
Just have a check Madhu.